scripts/cucumber.js

```diff
@@ -1,33 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Otherwise it considers `cross-spawn` as the library's save dependency:
```js
const spawn = require('cross-spawn');
```
So I disable this check here, as the entire script is for development purpose.